apps/src/applab/applab.js

```diff
@@ -806,9 +805,28 @@ function setupReduxSubscribers(store) {
     if (state.interfaceMode !== lastState.interfaceMode) {
       onInterfaceModeChange(state.interfaceMode);
     }
+
+    if (!lastState.runState || state.runState.isRunning !== lastState.runState.isRunning) {
+      Applab.onIsRunningChange(state.runState.isRunning);
```

Comment:
I was following a pattern.  I'll tear out the unused argument.  The handler for this just happens to depend on a utility method that calls `getState()`, so it doesn't actually need the new `isRunning` value from here because it gets it elsewhere.